lib/WebSocket.js

```diff
@@ -481,6 +481,25 @@ WebSocket.prototype.addEventListener = function(method, listener) {
   }
 };
 
+WebSocket.prototype.removeEventListener = function(method, listener) {
+  var listeners = this.listeners(method);
+  for (var i = 0; i < listeners.length; i++) {
+    if (listeners[i]._listener === listener) {
+      this.removeListener(method, listeners[i]);
+    }
+  }
+};
+
+WebSocket.prototype.hasEventListener = function(method, listener) {
```

Comment:
This is used only for testing right? If so can we remove it?
In the test we can use something like this:

```js
assert.strictEqual(ws.listeners().find((listener) => listener._listener === openListener), undefined);
```